### ISSUE TITLE
Queue page numbers can no longer be programmed into the URL

### ISSUE THREAD

**Issue by genghisken** [lukeshingles/atlasserver#5]
The new version of the paging is great, but if I have very large job, I can no longer "jump" back and forth through the pages by entering an offset.  It would also be great to jump straight to my most recently processed job, without having to hit "next" a thousand times!

**Comment by lukeshingles** [comment]
I could add "first" and "last" buttons perhaps, but in the mean time, couldn't you just click the "queue" link to get to the top?

**Comment by genghisken** [comment]
Yes - but my currently processing job is about 500 jobs in.  I have to click "next" at least 50 times to get to the current job!

**Comment by lukeshingles** [comment]
I see what you mean. I could add filter buttons for "finished", "queued", and "done"?

**Comment by lukeshingles** [comment]
I've added some filter buttons. Does this fix your use case?
